eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubProperties.java

```diff
@@ -14,19 +14,20 @@
     private final String path;
     private final Instant createdAt;
     private final String[] partitionIds;
-    private Instant propertyRetrievalTimeUtc;
 
-    EventHubProperties(
-            final String path,
-            final Instant createdAtUtc,
-            final String[] partitionIds,
-            final Instant propertyRetrievalTimeUtc) {
+    /**
+     * Creates a new instance of EventHubProperties.
+     *
+     * @param path The name of the Event Hub.
+     * @param createdAt The Instant when the Event Hub was created.
+     * @param partitionIds An array of identifiers for all the partitions in the Event Hub.
+     */
+    public EventHubProperties(String path, Instant createdAt, String[] partitionIds) {
```

Comment:
`String...` rather than `String[]` is more user friendly

Reply:
This isn't supposed to be public. I made it package-private. it's only created when returning information from a service call.